cloudbuild.yaml

```diff
@@ -34,16 +34,16 @@ steps:
   - mvn --no-transfer-progress -e install -Dlicense.skip=true -Dspotless.apply.skip=true
   waitFor: ['-']
 
-# - name: 'gcr.io/cloud-builders/curl'
-#   id: 'Upload coverage reports to codecov.io'
-#   entrypoint: bash
-#   args: ['-c', 'bash <(curl -s https://codecov.io/bash)']
-#   env:
-#     - 'VCS_COMMIT_ID=$COMMIT_SHA'
-#     - 'VCS_BRANCH_NAME=$BRANCH_NAME'
-#     - 'VCS_PULL_REQUEST=$_PR_NUMBER'
-#     - 'CI_BUILD_ID=$BUILD_ID'
-#     - 'CODECOV_TOKEN=$_CODECOV_TOKEN' # This token is generated when you setup your repo on codecov: https://docs.codecov.com/docs
+- name: 'gcr.io/cloud-builders/curl'
+  id: 'Upload coverage reports to codecov.io'
+  entrypoint: bash
+  args: ['-c', 'bash <(curl -f ../coverage/target/site/jacoco-aggregate/jacoco.xml -s https://codecov.io/bash)']
```

Comment:
What is this change supposed to do? The `-f` option of `curl` seems to be for "fail fast" and I think providing a filename after it is an error; or am I missing something?

Also suggestion: Let's drop `-s` such that we see curl errors in the e2e logs.